terraform/mesh_aws/s3_logs.tf

```diff
@@ -0,0 +1,66 @@
+resource "aws_s3_bucket" "s3logs" {
```

Comment:
should a configurable lifecycle rule be attached to this bucket, as it is essentially storing logs we don't know in this app

Reply:
It needs them for the triggering of the StepFunction from the S3 Event

But yeah, we should put a short retention on there that can be overwritten if anyone wants to keep them